stacks-common/src/types/mod.rs

```diff
@@ -58,11 +58,6 @@ pub trait Address: Clone + fmt::Debug + fmt::Display {
     fn is_burn(&self) -> bool;
 }
 
-pub const PEER_VERSION_EPOCH_1_0: u8 = 0x00;
```

Comment:
Is there a specific reason to get rid of these consts? I find it easier to grok the code with them. Is it just that the naming is off?

Reply:
There was an issue with the Clarinet devs not knowing which constants to be using, since they're duplicated here and in `stackslib/src/core/mod.rs`.

I suspect the answer is to move them all to `stacks-common/src/types/mod.rs` and then `pub use` them in `stackslib/src/core/mod.`

Reply:
> I suspect the answer is to move them all to stacks-common/src/types/mod.rs and then pub use them in stackslib/src/core/mod.rs

Yep, I agree.